test/scripts/jenkins_accessibility.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+trap 'node "$KIBANA_DIR/src/dev/failed_tests/cli"' EXIT
+
+node scripts/build --debug --oss;
+linuxBuild="$(find "$KIBANA_DIR/target" -name 'kibana-oss-*-linux-x86_64.tar.gz')"
+installDir="$PARENT_DIR/install/kibana"
+mkdir -p "$installDir"
+tar -xzf "$linuxBuild" -C "$installDir" --strip=1
+
+export TEST_BROWSER_HEADLESS=1
+
+checks-reporter-with-killswitch "Kibana accessibility tests" \
+  node scripts/functional_tests \
+    --debug --bail \
```

Comment:
Should we remove `--bail`? We can get all the failures at once otherwise.

CC @dmlemeshko